### PR TITLE
Add a scheduled task (for master) that fails if latest available SqlRace package is not installed

### PR DESCRIPTION
**Pull request by wei-mclarenapplied**

### CODE REVIEW

File: .github/workflows/Check-References.ps1 [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿. $PSScriptRoot"\Get-Package-Version.ps1"
+
+function Check-References([string] $RootDirectory, [string] $PackageName, [string[]] $VersionMustContain, [string[]] $VersionMustNotContain) {
+    # Find all of the packages.config files
+    $packageFiles = @(Get-ChildItem -Path $RootDirectory -Filter packages.config -Recurse | Select-Object -ExpandProperty "FullName")
+    
+	# The value we will return to indicate whether the references as valid or not
```
Comment (stanislav-domaratsky-mclaren) [inline .github/workflows/Check-References.ps1:7]: Misaligned

File: .github/workflows/Check-References.ps1 [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    $validationSuccessful = $true
+
+    foreach($packageFile in $packageFiles) { 
+        $version = Get-Package-Version -PackageFilePath $packageFile -PackageName $PackageName
+		
+		if($version -eq $null) {
```
Comment (stanislav-domaratsky-mclaren) [inline .github/workflows/Check-References.ps1:13]: Misaligned

**Review by stanislav-domaratsky-mclaren — CHANGES_REQUESTED** [review]
Please fix alignment. The rest looks good

File: .github/workflows/Check-References.ps1 (hunk 1 above)
Comment (wei-mclarenapplied) [reply .github/workflows/Check-References.ps1]: Bit strange that gh doesn't update the source file in this view after a push (just says 'outdated'), but it's fixed.

File: .github/workflows/Check-References.ps1 (hunk 2 above)
Comment (wei-mclarenapplied) [reply .github/workflows/Check-References.ps1]: as above

**Review by stanislav-domaratsky-mclaren — APPROVED** [review]

**Pull request merged**